app/graphql/types/user_type.rb

```diff
@@ -4,6 +4,7 @@ class UserType < Types::BaseObject
     field :email, String, null: false
     field :first_name, String, null: true
     field :last_name, String, null: true
+    field :avatar, ImageType, null: true
```

Comment:
```suggestion
    field :avatar_url, String, null: true
```

Does Shrine have URLs for uploaded files? It will be enough for this endpoint.